apps/main-app/src/containers/genePage/index.js

```diff
@@ -123,6 +126,10 @@ const GenePage = ({geneId}) => {
           <OrthologyFilteredTable geneId={data.id} />
         </Subsection>
 
+        <Subsection help={<OrthologyUserGuide />} title={PARALOGY}>
```

Comment:
We should create a different user guide even if its just a copy of the current one